src/entities/road.lua

```diff
@@ -3,18 +3,45 @@ local road = {}
 
 -- initialize road at (x, y) with each segment of given width and length
 function road:load(x, y, width, length)
-  self.roadjog = 40
+  self.roadPush = 1
+  self.roadSlide = self.roadPush/5
+  self.pivotLength = length * 0.6
+  sinLimit = (self.roadPush/2) / self.pivotLength
+  cosLimit = math.sqrt(self.pivotLength^2 - self.roadPush/2) / self.pivotLength
+  self.rotationLimit = math.pi - (math.abs(math.atan2(cosLimit, sinLimit)) * 2)
```

Comment:
So is the idea here that depending on how long the segment we "pave" each time is, we have to limit how tight the road segments can be angled in order to stay "smooth"?

Reply:
Commented. Let me know if this documentation is clear enough

Reply:
This is great, thanks!